Add new blog article: Why organizations are turning to decoupled authorization solutions | Cerbos (#40)
xNok/xNok: website/content/english/blogs/2023-11-21-why-organizations-are-turning-to-decoupled-authorization-solutions-cerbos.md

---

> title: "Why organizations are turning to decoupled authorization solutions | Cerbos"

Author: ```suggestion
title: "Why organizations are turning to decoupled authorization solutions"
```

date: 2023-11-21
draft: false
categories: ["Security"]

> tags: ["Authorization", "Decoupled Authorization"]

Author: ```suggestion
tags: ["Authorization"]
```

images:
 - https://cdn.builder.io/api/v1/image/assets/2517c53399d846eea97f6c5ae8804b7f/842076d3c54c4c2789a2db691d28afe8
---

> Original Article : [Read on Cerbos](https://www.cerbos.dev/blog/why-organizations-are-turning-to-decoupled-authorization-solutions)

## Summary

> This article explores the increasing adoption of **decoupled authorization solutions** by organizations seeking to enhance productivity, security, and reliability. It contrasts traditional authorization methods with the benefits of decoupling authorization to an external service, which enables developers to concentrate on implementing business logic. The advantages of decoupled authorization include enhanced flexibility, customization, cost reduction, and the prevention of errors. It also addresses the pattern towards outsourcing security functions and highlights the risks associated with **decoupled authorization**, such as vendor lock-in, availability, scalability, and latency. 

Author: ```suggestion
This article explores the increasing adoption of decoupled authorization solutions by organizations seeking to enhance productivity, security, and reliability. In contrast with traditional authorization methods, delegating authorization to an external service enables developers to concentrate on implementing business logic. 

The advantages of decoupled authorization include enhanced flexibility, customization, and error prevention. It also addresses the pattern of outsourcing security functions, which become easier to review, audit and delegate to subject matter experts.
```

Key topics discussed:

*   **Traditional vs. Decoupled Authorization:** Understand the differences between traditional and decoupled authorization methods.

> *   **Advantages of Decoupled Authorization:** Discover the benefits of flexibility, customization, and cost reduction.

Author: ```suggestion
```

*   **Risks of Decoupled Authorization:** Identify potential risks like vendor lock-in and performance considerations.

> *   **Outsourcing Security Functions:** Learn about the increasing trend of outsourcing security functions, including authentication, encryption, and threat management.

Author: ```suggestion
*   **Outsourcing Security Functions:** Learn about this increasing trend of outsourcing security functions, including authentication, encryption, and threat management.
```
